package/com.unity.formats.usd/Runtime/InitUsd.cs

```diff
@@ -12,10 +12,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+using System.Diagnostics;
 using System.Runtime.CompilerServices;
-using UnityEngine;
 using USD.NET;
 using USD.NET.Unity;
+using Debug = UnityEngine.Debug;
```

Comment:
Not used?

Reply:
Yep, as above this seems to be some automatic thing rider has done to replace UnityEngine. I'll put it back :)